app/src/main/java/ch/epfl/cs311/wanderwave/di/DatabaseModule.kt

```diff
@@ -18,6 +20,16 @@ object DatabaseModule {
   @Singleton
   fun provideAppDatabase(@ApplicationContext context: Context): AppDatabase {
     return Room.databaseBuilder(context.applicationContext, AppDatabase::class.java, "app_database")
+        .addMigrations(MIGRATION_1_2)
         .build()
   }
+
+  // Migration from version 1 to version 2: adding the TrackRecord table
```

Comment:
What is this for?

Reply:
I think this is a good think in general, it always creates corruption problems that we don't have time to solve, and this is the best way of solving the problem.